packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -278,6 +278,23 @@ export class Account {
     return await this.client.call('accounttoutxos', [from, payload, options.utxos], 'number')
   }
 
+  async listAccountHistory (): Promise<AccountHistory[]>;
+
+  async listAccountHistory (
+    owner: OwnerType | string,
+    options: Omit<AccountHistoryOptions, 'no_rewards'> & {no_rewards: false}
+  ): Promise<AccountHistory[]>;
+
+  async listAccountHistory (
+    owner: OwnerType | string,
+    options: Omit<AccountHistoryOptions, 'no_rewards'> & {no_rewards: true}
```

Comment:
@surangap can confirm everything else should come with txid when `no_rewards` true?
The flag also handle dToken pair rewards and permission right?

Reply:
`no_rewards` controls the pool rewards. so yeah `no_rewards = true` should give all with txids.
when calculating pool rewards, it iterates through all pools. so dtoken pools are also included.

Reply:
maybe can add a testcase for👆 if not there already

Reply:
👍 I don't think there is any test with rewards distributing in action. See https://github.com/DeFiCh/jellyfish/blob/main/apps/playground-api/src/setups/setup.dex.ts#L212-L215 for the setup required @cwkang1998